ResultTests/ResultTests.swift

```diff
@@ -31,10 +31,21 @@ final class ResultTests: XCTestCase {
 	}
 
 	func testErrorsIncludeTheCallingFunction() {
-		let function = __FUNCTION__
-		XCTAssertEqual(Result<(), NSError>.error().function ?? "", function)
+//		let function = __FUNCTION__
+//		XCTAssertEqual(Result<(), NSError>.error().function ?? "", function)
```

Comment:
funny turn out it can get it to crash only when inside the tests target, if same code is added to result itself:

```
func test() {
	let function = __FUNCTION__
	let expected = Result<(), NSError>.error().function ?? ""
	if function == expected {
		
	}
}
```

It doesn't crash.